### assembly with nanopore_assembly.sh version 2 ###
# The illumina fastq files were the files produced by 1_trimming.sh

# B. japonicum group #

./nanopore_assembly.sh /datadisk1/Nanopore_data/20220223-Bradyrhizobium-mutants/Basecalled_data/Set_1/barcode01_reads.fastq.gz CPAC15/fastq_files/566_S1_L001_R1_bacterial.fastq.gz CPAC15/fastq_files/566_S1_L001_R2_bacterial.fastq.gz CPAC15/17nanopore_assembly 16 16
./nanopore_assembly.sh /datadisk1/Nanopore_data/20220223-Bradyrhizobium-mutants/Basecalled_data/Set_1/barcode03_reads.fastq.gz CPAC15/fastq_files/S370_S4_L001_R1_bacterial.fastq.gz CPAC15/fastq_files/S370_S4_L001_R2_bacterial.fastq.gz CPAC15/23nanopore_assembly 16 16
./nanopore_assembly.sh /datadisk1/Nanopore_data/20220223-Bradyrhizobium-mutants/Basecalled_data/Set_1/barcode04_reads.fastq.gz CPAC15/fastq_files/24_S1_L001_R1_bacterial.fastq.gz CPAC15/fastq_files/24_S1_L001_R2_bacterial.fastq.gz CPAC15/24nanopore_assembly 16 16
./nanopore_assembly.sh /datadisk1/Nanopore_data/20220223-Bradyrhizobium-mutants/Basecalled_data/Set_1/barcode05_reads.fastq.gz CPAC15/fastq_files/29_S2_L001_R1_bacterial.fastq.gz CPAC15/fastq_files/29_S2_L001_R2_bacterial.fastq.gz CPAC15/29nanopore_assembly 16 16
./nanopore_assembly.sh /datadisk1/Nanopore_data/20220223-Bradyrhizobium-mutants/Basecalled_data/Set_1/barcode06_reads.fastq.gz CPAC15/fastq_files/31_S3_L001_R1_bacterial.fastq.gz CPAC15/fastq_files/31_S3_L001_R2_bacterial.fastq.gz CPAC15/31nanopore_assembly 16 16
./nanopore_assembly.sh /datadisk1/Nanopore_data/20220223-Bradyrhizobium-mutants/Basecalled_data/Set_1/barcode07_reads.fastq.gz CPAC15/fastq_files/34_S4_L001_R1_bacterial.fastq.gz CPAC15/fastq_files/34_S4_L001_R2_bacterial.fastq.gz CPAC15/34nanopore_assembly 16 16
./nanopore_assembly.sh /datadisk1/Nanopore_data/20220223-Bradyrhizobium-mutants/Basecalled_data/Set_1/barcode08_reads.fastq.gz CPAC15/fastq_files/S204_S2_L001_R1_bacterial.fastq.gz CPAC15/fastq_files/S204_S2_L001_R2_bacterial.fastq.gz CPAC15/38nanopore_assembly 16 16

# B. diazoefficiens group #

./nanopore_assembly.sh /datadisk1/Nanopore_data/20220223-Bradyrhizobium-mutants/Basecalled_data/Set_2/barcode10_reads.fastq.gz CPAC7/fastq_files/586_S5_L001_R1_bacterial.fastq.gz CPAC7/fastq_files/586_S5_L001_R2_bacterial.fastq.gz CPAC7/10nanopore_assembly2 16 16
./nanopore_assembly.sh /datadisk1/Nanopore_data/20220223-Bradyrhizobium-mutants/Basecalled_data/Set_2/barcode11_reads.fastq.gz CPAC7/fastq_files/CPAC390_S6_L001_R1_bacterial.fastq.gz CPAC7/fastq_files/CPAC390_S6_L001_R2_bacterial.fastq.gz CPAC7/104nanopore_assembly 16 16
./nanopore_assembly.sh /datadisk1/Nanopore_data/20220223-Bradyrhizobium-mutants/Basecalled_data/Set_2/barcode12_reads.fastq.gz CPAC7/fastq_files/CPAC392_S7_L001_R1_bacterial.fastq.gz CPAC7/fastq_files/CPAC392_S7_L001_R2_bacterial.fastq.gz CPAC7/105nanopore_assembly 16 16
./nanopore_assembly.sh /datadisk1/Nanopore_data/20220223-Bradyrhizobium-mutants/Basecalled_data/Set_2/barcode01_reads.fastq.gz CPAC7/fastq_files/106_S5_L001_R1_bacterial.fastq.gz CPAC7/fastq_files/106_S5_L001_R2_bacterial.fastq.gz CPAC7/106nanopore_assembly 16 16
./nanopore_assembly.sh /datadisk1/Nanopore_data/20220223-Bradyrhizobium-mutants/Basecalled_data/Set_2/barcode02_reads.fastq.gz CPAC7/fastq_files/CPAC394_S8_L001_R1_bacterial.fastq.gz CPAC7/fastq_files/CPAC394_S8_L001_R2_bacterial.fastq.gz CPAC7/107nanopore_assembly 16 16
./nanopore_assembly.sh /datadisk1/Nanopore_data/20220223-Bradyrhizobium-mutants/Basecalled_data/Set_2/barcode03_reads.fastq.gz CPAC7/fastq_files/108_S6_L001_R1_bacterial.fastq.gz CPAC7/fastq_files/108_S6_L001_R2_bacterial.fastq.gz CPAC7/108nanopore_assembly2 16 16
./nanopore_assembly.sh /datadisk1/Nanopore_data/20220223-Bradyrhizobium-mutants/Basecalled_data/Set_2/barcode04_reads.fastq.gz CPAC7/fastq_files/109_S7_L001_R1_bacterial.fastq.gz CPAC7/fastq_files/109_S7_L001_R2_bacterial.fastq.gz CPAC7/109nanopore_assembly 16 16
./nanopore_assembly.sh /datadisk1/Nanopore_data/20220223-Bradyrhizobium-mutants/Basecalled_data/Set_2/barcode05_reads.fastq.gz CPAC7/fastq_files/110_S8_L001_R1_bacterial.fastq.gz CPAC7/fastq_files/110_S8_L001_R2_bacterial.fastq.gz CPAC7/110nanopore_assembly 16 16
nanopore_assembly.sh /datadisk1/Sequencing_data/Nanopore_data/20220223-Bradyrhizobium-mutants/Basecalled_data/Set_1/barcode09_reads.fastq.gz fastq_files/CNPSO6_S5_L001_R1_bacterial.fastq.gz fastq_files/CNPSO6_S5_L001_R2_bacterial.fastq.gz cpac7nanopore_assembly 16 16

### Assembly with unicycler version 0.5.0 ###

# CNPSo 22 #

unicycler -1 Documents/nanopore_assembly/CPAC15/fastq_files/S340_S3_L001_R1_bacterial.fastq.gz -2 ../nanopore_assembly/CPAC15/fastq_files/S340_S3_L001_R2_bacterial.fastq.gz -l /datadisk1/Sequencing_data/Nanopore_data/20220223-Bradyrhizobium-mutants/Basecalled_data/Set_1/barcode02_reads.fastq.gz -o 22unicycler -t 16


### RagTag version 2.1.0 ###

# CNPSo 22 #

ragtag.py patch -t 16 22nanopore_assembly/overlap_removal_2/22overlap2.fasta 22unicycler/22unicycler.fasta -o 22ragtag-April27


### Manually rename final assemblies as Xfinal.fasta, where X is a number corresponding to the strain, e.g., 17final.fasta for CNPSo 17 ###
